client/app/bundles/shared/components/Resource/Resources.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import css from './Resources.scss';
+import Tag from '../Tag';
+
+type Props = {
+  resourcesObj?: Object,
+  tagged?: boolean,
+};
+
+export default class Resources extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  render() {
+    const { resourcesObj, tagged } = this.props;
+    const resources = tagged ? resourcesObj.tags.map((tag, index) =>
+      <Tag normal label={tag} key={index} />,
```

Comment:
I know this is not the same component used, but for `<Tag normal />`s, the background color is now off for some reason. Could you tweak it so that it is `rgba(255, 255, 255, 0.1)`?